generate_pipeline.py

```diff
@@ -0,0 +1,57 @@
+from jinja2 import Template
+from pathlib import Path
+
+ingests = []
+
+ingest_dirs = [dir.parent for dir in Path("./monarch_ingest").rglob("metadata.yaml")]
+
+for ingest_dir in ingest_dirs:
+    source_ingests = [ingest_file for ingest_file in list(ingest_dir.rglob("*.yaml"))
+                      if not ingest_file.match("metadata.yaml")]
+    for ingest in source_ingests:
+        ingests.append({
+            'source':str(ingest.parent.stem),
+            'source_file':str(ingest.name),
+            'source_file_stem': str(ingest.stem),
+            'ingest_method': 'ingest_' + str(ingest.parent.stem) + '_' + str(ingest.stem) + '()'
+        })
+
+ingest_methods = ', '.join([ingest['ingest_method'] for ingest in ingests])
+
+
+template = """
```

Comment:
I'm unfamiliar with jinja but is there a way to put this is another file where we get static checks on the non-templated bits?